UnitTests/BTDropInUICustomizationTests.swift

```diff
@@ -86,6 +88,61 @@ class BTDropInUICustomizationTests: XCTestCase {
             XCTAssertNil(uiCustomization.fontFamily)
             XCTAssertNil(uiCustomization.boldFontFamily)
             XCTAssertNil(uiCustomization.navigationBarTitleTextColor)
+            XCTAssertFalse(uiCustomization.disableDynamicType)
         }
     }
+
+    func testCustomFonts_withDynamicTypeDisabled() {
```

Comment:
I think these tests should actually go in `BTUIKAppearance+DropInTests.swift` since they're testing the `configure` method on `BTUIKAppearance+DropIn`.